schematic/store/synapse.py

```diff
@@ -291,8 +291,19 @@ def query_fileview(
                 self.storageFileviewTable = self.syn.tableQuery(
                     query=self.fileview_query,
                 ).asDataFrame()
-            except SynapseHTTPError:
-                raise AccessCredentialsError(self.storageFileview)
+            except SynapseHTTPError as exc:
+                exception_text = str(exc)
+                if "Unknown column path" in exception_text:
+                    raise ValueError(
+                        "The path column has not been added to the fileview. Please make sure that the fileview is up to date. You can add the path column to the fileview by follwing the instructions in the validation rules documentation."
+                    ) from exc
+                elif "Unknown column" in exception_text:
+                    missing_column = exception_text.split("Unknown column ")[-1]
+                    raise ValueError(
+                        f"The columns {missing_column} specified in the query do not exist in the fileview. Please make sure that the column names are correct and that all expected columns have been added to the fileview."
+                    ) from exc
+                else:
+                    raise AccessCredentialsError(self.storageFileview)
```

Comment:
Does this also need the `from exc` clause?

Reply:
I was trying to be more complete and include the context from synapse, but if you think it's not necessary then I can remove @BryanFauble

Reply:
What I meant was, does this line:
`raise AccessCredentialsError(self.storageFileview)`
need to be
`raise AccessCredentialsError(self.storageFileview) from exc`
?

exception handling in python is really un-intuitive, and I'm not sure if this is needed to properly maintain the context from the original exception. This is what you have doing in the 2 ValueErrors

Reply:
@BryanFauble that makes sense